webapp/channels/src/components/admin_console/group_settings/group_details/group_users_row.tsx

```diff
@@ -16,39 +16,39 @@ type Props = {
     lastPictureUpdate: number;
 };
 
-export default class GroupUsersRow extends React.PureComponent<Props> {
-    render = () => {
-        return (
-            <div className='group-users-row'>
-                <Avatar
-                    username={this.props.username}
-                    url={Client4.getProfilePictureUrl(
-                        this.props.userId,
-                        this.props.lastPictureUpdate,
-                    )}
-                    size='lg'
-                />
-                <div className='user-data'>
-                    <div className='name-row'>
-                        <span className='username'>
-                            {'@' + this.props.username}
-                        </span>
-                        {'-'}
-                        <span className='display-name'>
-                            {this.props.displayName}
-                        </span>
-                    </div>
-                    <div>
-                        <span className='email-label'>
-                            <FormattedMessage
-                                id='admin.group_settings.group_details.group_users.email'
-                                defaultMessage='Email:'
-                            />
-                        </span>
-                        <span className='email'>{this.props.email}</span>
-                    </div>
+export default function GroupUsersRow(props: Props) {
```

Comment:
Just to follow the same pattern we are using in the rest of the product, could you:
- Write this as an arrow function
- Expand the props in the arguments list
- Export it at the end of the file

Also, since it is a pure component, it should use a `React.memo`.

So... something like this:
```suggestion
const GroupUsersRow= ({
    username,
    displayName,
    ...
}: Props) {
    ...
}

export default React.memo(GroupUsersRow);
```